### PR TITLE
feat: add sitemap plugin

### LINKED ISSUE #522: [NG] Add sitemap plugin
**Desired Behavior**
Add and configure the [gatsby plugin for sitemap](https://www.gatsbyjs.org/packages/gatsby-plugin-sitemap/?=sitemap) in [layer5-ng](https://github.com/layer5io/layer5/tree/layer5-ng). Related to #481

### PR DESCRIPTION
**Pull request by Tanuj22**
Signed-off-by: Tanuj22 <tanujagarwal22@gmail.com>

**Description**

This PR fixes #522

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ module.exports = {
     title: `Layer5 - The Service Mesh Company`,
     description: `Making service meshes available to the rest of us. Open source software for management of service meshes. Allowing developers to focus on business logic, not infrastructure concerns. Empowering operators to confidentally run modern infrastructure.`,
     author: `Layer5 Authors`,
+    siteUrl: `https://layer5ng.netlify.app/`,
```
Comment (Tanuj22) [inline gatsby-config.js:6]: URL has to be changed later.

**Comment by Tanuj22** [comment]
The sitemap is available [here](https://deploy-preview-630--layer5ng.netlify.app/sitemap.xml)

File: gatsby-config.js (hunk 1 above)
Comment (leecalcote) [reply gatsby-config.js:6]: Good to note

**Comment by Nikhil-Ladha** [comment]
@Tanuj22 please do open one issue for that so, that we can remember it in future. Also, add it in `To-do` after creating the issue.

**Pull request merged**